source/developers-guide/shopware-5-cli-commands/index.md

```diff
@@ -109,3 +109,54 @@ EOF
 ```
 
 As you can see, a Shopware CLI command is very similar to a Symfony 3 command. You can use any of the features provided by the Symfony Console component, like you would in Symfony 3 itself.
+
+## Add completion for commands
+
+Shopware uses [stecman/symfony-console-completion](https://github.com/stecman/symfony-console-completion) to add completion features you already know from any CLI.
+This composer package automatically adds completion to any command about the option names.
```

Comment:
do you mean `... via the option names` ? otherwise I don't understand this sentence :blush:

Reply:
I wanted to say that by default this package can only expand option names. I think about a better sentence.